p04-mui-quotes/src/Components/Quote/Tags/ListTags.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { Stack } from '@mui/material';
+import Paper from '@mui/material/Paper';
+import { styled } from '@mui/material/styles';
+import Divider from '@mui/material/Divider';
+
+const Item = styled(Paper)(({ theme }) => ({
```

Comment:
Item may be on a separated file with all the styles for quotes or listTags styles